dojo/tools/awssecurityhub/parser.py

```diff
@@ -20,70 +21,87 @@ def get_findings(self, filehandle, test):
     def get_items(self, tree, test):
         items = {}
         # DefectDojo/django-DefectDojo/issues/2780
-        findings = tree.get("Findings", tree.get("findings", None))
+        findings = tree.get('Findings', tree.get('findings', None))
 
         if not findings:
             return list()
 
         for node in findings:
             item = get_item(node, test)
-            key = node["Id"]
+            key = node['Id']
             items[key] = item
 
         return list(items.values())
 
 
 def get_item(finding, test):
-    finding_id = finding.get("Id", "")
-    title = finding.get("Title", "")
-    severity = (
-        finding.get("Severity", {}).get("Label", "INFORMATIONAL").title()
-    )
-    description = finding.get("Description", "")
-    resources = finding.get("Resources", "")
-    resource_id = resources[0]["Id"].split(":")[-1]
-    mitigation = (
-        finding.get("Remediation", {})
-        .get("Recommendation", {})
-        .get("Text", "")
-    )
-    references = (
-        finding.get("Remediation", {}).get("Recommendation", {}).get("Url")
-    )
-    false_p = False
+    aws_scanner_type = finding.get('ProductFields', {}).get('aws/securityhub/ProductName', "")
```

Comment:
Please use double quotes consistently

Reply:
thanks for your input. I just changed it with latest commit.